files/en-us/web/api/node/comparedocumentposition/index.html

```diff
@@ -97,9 +110,9 @@ <h2 id="Example">Example</h2>
 
 <div class="note">
   <p><strong>Note:</strong> Because the result returned by
-    <code>compareDocumentPosition()</code> is a bitmask, the <a
-      href="/en-US/docs/JavaScript/Reference/Operators/Bitwise_Operators">bitwise AND
-      operator</a> must be used for meaningful results.</p>
+    <code>compareDocumentPosition()</code> is a bitmask, the
+    <a href="/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators">bitwise AND operator</a>
```

Comment:
```suggestion
    <a href="/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_AND">bitwise AND operator</a>
```

(Second attempt, sorry)